test/integration/patients/sidebar/action-sidebar.js

```diff
@@ -1037,6 +1048,64 @@ context('action sidebar', function() {
       .should('not.exist');
   });
 
+  specify('action attachments - uploads not allowed for org', function() {
+    cy
+      .routesForPatientAction()
+      .routeSettings(fx => {
+        fx.data.push({ id: 'upload_attachments', attributes: { value: false } });
+
+        return fx;
+      })
+      .routeAction(fx => {
+        fx.data = {
+          id: '1',
+          attributes: {
+            name: 'Test Action',
+            allowed_uploads: ['pdf'],
+          },
+          relationships: {
+            owner: { data: { id: '11111', type: 'clinicians' } },
+            files: { data: [{ id: '1' }] },
+          },
+        };
+
+        return fx;
+      })
+      .routeActionFiles(fx => {
+        fx.data = [
+          {
+            id: '1',
+            attributes: {
+              path: 'patients/1/HRA.pdf',
+              created_at: '2019-08-24T14:15:22Z',
+            },
+            meta: {
+              view: 'https://www.bucket_name.s3.amazonaws.com/patients/1/view/HRA.pdf',
+              download: 'https://www.bucket_name.s3.amazonaws.com/patients/1/download/HRA.pdf',
+            },
+          },
+        ];
+
+        return fx;
+      })
+
+      .visit('/patient/1/action/1')
+      .wait('@routeAction')
+      .wait('@routeActionFiles');
+
+    cy
+      .get('.sidebar')
+      .find('[data-attachments-files-region]')
+      .children()
+      .should('have.length', 1);
+
+    cy
+      .get('.sidebar')
+      .find('[data-attachments-region]')
+      .find('.js-add')
+      .should('not.exist');
+  });
+
```

Comment:
I need to test these two scenarios:

1. Upload attachment button is hidden because the `upload_attachments` org setting is set to `false` (or doesn't exist).
2. Upload attachment button is hidden because the `allowed_uploads: []` is empty or non-existent for the action.

These can't be tested together in the same test. Since if one or the either is `false`, the button will be hidden.

So I split them into two separate ones.